plugins/kotlin/idea/tests/testData/quickfix/addStarProjections/checkType/array.kt

```diff
@@ -0,0 +1,9 @@
+// "Change type arguments to <*, *>" "false"
```

Comment:
`Change type arguments to <*>`

Reply:
[add46db](https://github.com/JetBrains/intellij-community/pull/2083/commits/add46dbcca6ac0dc9c892ef1e080ad09a43b5283)